themes/bootstrap5/templates/ContentBlock/FacetList.phtml

```diff
@@ -14,6 +14,7 @@
       <div class="home-facet <?=$this->escapeHtmlAttr($field) ?><?=$isHierarchy ? ' facet-tree' : ''?>">
         <h2><?=$labelHeading?></h2>
         <div class="home-facet-container">
+          <?php /* removed the 'callnumber-first' coding here because 'callnumber-first' is set on the 'home-facet' container, so columns can be set using SCSS */ ?>
```

Comment:
I don't think we need to keep this comment here; if we're removing the hard-coded behavior, we don't need to keep a comment about its removal forever. I think people can rely on git to figure out the history if they're curious about the change.


```suggestion
```